pkg/handlers/v1/ec2.go

```diff
@@ -202,8 +202,8 @@ func sliceDiff(a, b []string) []string {
 // extract network interface information from an ec2 configuration
 func extractEC2NetworkInfo(config *ec2Configuration) Change {
 	change := Change{}
-	for _, ni := range config.NetworkInterfaces {
-		private, public, dns := extractNetworkInterfaceInfo(&ni)
+	for i := range config.NetworkInterfaces {
```

Comment:
as per https://stackoverflow.com/a/62486667 it is safer to use an index here instead of a pointer.